data/operators/amenity/library.json

```diff
@@ -873,6 +873,62 @@
         "operator:wikidata": "Q6458744"
       }
     },
+    {
+      "displayName": "London Borough of Hounslow",
+      "id": "londonboroughofhounslow-d8f64f",
+      "locationSet": {
+        "include": ["gb-lon.geojson"]
+      },
+      "tags": {
+        "amenity": "library",
+        "operator": "London Borough of Hounslow",
+        "operator:wikidata": "Q214162",
+        "ref:isil": "GB-UkHnHL"
```

Comment:
I'm curious about `ref:isil` and `ref:ISIL` ? Both of them have many element using according to taginfo:

![图片](https://user-images.githubusercontent.com/42690037/172042992-090651aa-e2be-4682-b3c5-70ed25f53cf7.png)

[OSM Wiki](https://wiki.openstreetmap.org/wiki/Key%3Aref%3Aisil) said this This key references a library by its [ International Standard Identifier for Libraries and Related Organisations](https://en.wikipedia.org/wiki/en:International_Standard_Identifier_for_Libraries_and_Related_Organisations), I want to know is every branch amenity of this library network share the same ISIL?

---

By the way, `"id"` is optional, not mandatory when you add a new entry because it can be auto generated by formatter program. You can remove it or wait for maintaner run a `npm run build` script to format it.

Reply:
> I want to know is every branch amenity of this library network share the same ISIL?

At least in the UK, the British Library assigns an ISIL to organisations of libraries, rather than each individual branch - see: https://www.bl.uk/help/get-an-isil-for-your-library. Therefore, all the libraries run by the same borough will have the same ISIL as they are part of the same organisation.

> By the way, "id" is optional, not mandatory when you add a new entry because it can be auto generated by formatter program.

Yes, I ran `npm run build` because the wiki says I could do so if possible, but I will remove it if it is simpler.